test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -5215,6 +5227,92 @@ public void testModifiedFolder() throws Exception {
         new Checkpoint(FEB_1970, folderId));
   }
 
+  private void testModifiedDocumentsQuery(String query, List<Record> expected)
+      throws Exception {
+    String parentId = FOLDER.pad("FFF1");
+    String parentFolder = "/FFF1";
+    insertFolder(EPOCH_1970, parentId, parentFolder);
+    String folderId = FOLDER.pad("FFF2");
+    String folder = "/FFF1/FFF2";
+    insertFolder(FEB_1970, folderId, folder);
+    insertDocument(FEB_1970, DOCUMENT.pad("aaa"), folder + "/aaa", folderId);
+    insertDocument(MAR_1970, DOCUMENT.pad("bbb"), folder + "/bbb", folderId);
+
+    checkModifiedDocIdsPushed(
```

Comment:
I think I would just move this to a baseline test (e.g., testModifiedDocumentsQuery_Default or something). No need to run it every time. If you really prefer not setting documentum.modifiedDocumentsQuery (even to ""), then you could take the config map as a first argument instead of the query.